### DISCUSSION TITLE
Clarification: Relationships between Balance & Piggy Banks/Budget/Bills

### DISCUSSION THREAD

**Discussion by lairdman** [firefly-iii/firefly-iii#5235] [Firefly III] [answered]
Is there a relationship between balance & piggy banks, budgets & bills? I think the below is correct.

Budgets seems to be unrelated to the account balance. I'll admit I struggle a little with this, but I can get on board; I can plan to spend more money than I have by running up the credit card or going negative balance. 

Piggy banks are limited by the account balance and  budgets have no impact them. They're savings goals; you can't save money don't have. I'd live with budgetting better if you couldn't save more than your balance less planned spend (budget).  

Bills, likely are unrelated to either; the bills come regardless of whether you have the money.

**Comment by JC5** [comment, marked as answer]
Sounds correct to me!
